Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -2508,7 +2506,7 @@ The following are tips for handling issues with mirroring incidents between Serv
+| Ticket is not closing. | Verify the integration instance is configured correctly for the *Mirrored XSOAR Ticket closure method* or *Close Mirrored ServiceNow Ticket* fields.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Same as above, just need to fix the param naming

Reply:
Done.